src/styles/search__label.css

```diff
@@ -0,0 +1,32 @@
+.search__label {
+  position: relative;
+  display: block;
+  width: 100%;
+  height: 100%;
```

Comment:
Avoid fixing the container size with 100% height unless there is a specific requirement. It's better to let the content size dictate it to avoid overflow or accidental scroll bars.